cloud/pkg/edgecontroller/controller/downstream.go

```diff
@@ -165,7 +165,7 @@ func (dc *DownstreamController) syncSecret() {
 			}
 
 			nodes := dc.lc.SecretNodes(secret.Namespace, secret.Name)
-			klog.Infof("there are %d nodes need to sync secret, operation: %s", len(nodes), e.Type)
+			klog.V(4).Infof("there are %d nodes need to sync secret, operation: %s", len(nodes), e.Type)
```

Comment:
changed for debug purpose?

Reply:
This results in too many useless debug logs